oat-sa/tao-test-runner-qti-fe
# [TAO-9478] Create TTS component

Related to : https://oat-sa.atlassian.net/browse/TAO-9481, https://oat-sa.atlassian.net/browse/TAO-9482
 
Implement playback mods for TTS component
  
#### How to test
 
- Run unit test for the tts component by `test:keepAlive` package.json script
- Open test page in browser and check the component
